lib/hanzi/characters.map

```diff
@@ -13806,7 +13803,7 @@
 嘶 a sī
 嘶 p sī
 嘷 p háo
-嘸 a fǔ m̄
```

Comment:
Oh no, this m isn't a single unicode code point like all the other accented "vowels".
It's U+006D and U+0304

Reply:
https://en.wiktionary.org/wiki/%E5%98%B8

Reply:
I'm working on the new version of `from_marked` which should be able to handle this, so we can parse this situation. I should have a few commits ready soon.